### DISCUSSION TITLE
Hubitat c7 with Homebridge v2

### DISCUSSION THREAD

**Discussion by eweber911** [tonesto7/homebridge-hubitat-tonesto7#135] [General]
Hello all, 

Sorry to post but I've spent HOURS trying to figure this out and it's driving me crazy!

Running hubitat c7 with homebridge v2 plugin.

My problem is, I can't find a homekit code or any way to add this "hub" into the homekit app as a hub.  Not code to add, no nothing that I can find.  What on earth am I doing wrong?

Thanks in advance.

**Comment by jtp10181** [comment]
Are you running a homebridge server instance? When you log into the server interface it gives you the QR code to add it to HomeKit. The plugin on Hubitat talks to a separate Homebridge-Hubitat plugin that must be running on a homebridge server.
